src/test/java/fi/aalto/cs/apluscourses/integration/ApiTest.java

```diff
@@ -14,7 +14,7 @@
 
   //  For this to work the 'CI=true' environment variable is added to .travis.yml
   @ClassRule
-  public static final EnvironmentChecker checker = new EnvironmentChecker("CI");
+  public static final EnvironmentChecker checker = new EnvironmentChecker("INTEGRATION");
```

Comment:
This env-var is explicitly set for PRs but not pushes. Intended?

Reply:
It's so the push pipelines can finish fast without having to start the local O1 instance